docs/conf.py

```diff
@@ -19,8 +19,12 @@
 #
 import os
 import sys
+from configparser import ConfigParser
 
-from __af_version__ import full_version
+
+config = ConfigParser()
+config.read(os.path.abspath("setup.cfg"))
+full_version = config.get("metadata", "version")
```

Comment:
Fetching the full_version from setup config is good, what about the variables `version` and `release` ? Aren't they used anywhere else other than their declaration in the project ?

Reply:
No, they don't. Also, there is no need to update the version of the package in multiple files at the same time if it can be changed once in `setup.cfg`.

Reply:
Ok, just wanted to check if looked for those in the code base as I didn't run your changes on any system yet.

Reply:
I'm not sure about the correctness of my checks for docs. How do you compile the docs? What command do you use for it?